addons/website_blog/static/src/js/website_blog.inline.discussion.js

```diff
@@ -50,6 +51,21 @@ var InlineDiscussion = core.Class.extend({
                         'marginLeft': "+=250px"
                     });
                 }
+            } else if ($(event.target).hasClass('discussion-link')) {
```

Comment:
@Gorash  which two conditions ?, self.hide_discussion() resides inside else if ($(event.target).hasClass('discussion-link')), would you please elaborate more, which two condition you want to concatenate.